pkg/kubectl/cmd/delete/delete_flags.go

```diff
@@ -156,9 +156,10 @@ func NewDeleteCommandFlags(usage string) *DeleteFlags {
 
 	filenames := []string{}
 	recursive := false
+	kustomize := []string{}
 
 	return &DeleteFlags{
-		FileNameFlags: &genericclioptions.FileNameFlags{Usage: usage, Filenames: &filenames, Recursive: &recursive},
+		FileNameFlags: &genericclioptions.FileNameFlags{Usage: usage, Filenames: &filenames, Kustomize: &kustomize, Recursive: &recursive},
```

Comment:
Why doesn't this use the helpers.go?  We should at least document it.

Reply:
`DeleteFlags` composes common flag struct used for commands requiring deletion logic, such as `kubectl delete` and `kubectl apply`. It contains `FileNameFlags`, which is then converted to `FileNameOptions`.

While in other subcommand options, `FileNameOptions` is directly used instead of `FileNameFlags`. helpers.go contains functions to add `-k` and `-f` for `FileNameOptions`, but not for `FileNameFlags`.

In the future, I think the logic of adding flags should be moved to `cli-runtime`. Do we convert all current `FileNameOptions` to `FileNameFlags` or keep both of them? I'm not sure. @juanvallejo What do you think?

Reply:
Please add a comment to this code explaining why it doesn't use `helpers.go`

Reply:
How about 
```
// Not using helpers.go it since provides function to add '-k' for FileNameOptions, but not FileNameFlags
```